core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -77,6 +79,22 @@ var ExplorationPlayerPage = function() {
     nextCardButton.click();
   };
 
+  this.reportExplorationButton = function() {
```

Comment:
change the name to `reportExploration` since you aren't just using a button.

Reply:
Done!